src/provider/store-data.spec.ts

```diff
@@ -11,20 +11,24 @@ import {
     resetDB,
     mediator,
     backend,
-    unverifiedProvider,
+    verifiedProvider,
 } from "../testing/fixtures"
+import { VanellusError } from '../errors'
 
 describe("Provider.storeData()", function () {
     it("we should be able to store provider data", async function () {
         const be = backend()
         const keys = await adminKeys()
         await resetDB(be, keys)
         const med = await mediator(be, keys)
-        const up = await unverifiedProvider(be, keys)
+        
+        const up = await verifiedProvider(be, keys, med)
```

Comment:
`storeData()` has to be possible for an unverified provider since this step uploads the provider data that is later verified by the mediator. For verified providers this function is unimportant.

```suggestion
        const up = await unverifiedProvider(be)
```